kolibri/core/assets/src/views/ImmersiveToolbar.vue

```diff
@@ -8,11 +8,6 @@
       :showIcon="showIcon"
       :style="{
         height: topBarHeight + 'px',
-        position: 'fixed',
```

Comment:
I looked for places where this component was used, and it was only used in `ImmersivePage` and `CoreBase` and in both it would be wrapped by the `ScrollingHeader` component, so there should be no unintended changes.